react-with-graphql-tutorial.md

```diff
@@ -1245,19 +1246,21 @@ const getIssuesOfRepository = path => {
 };
 {{< /highlight >}}
 
-That's it. Finally the query takes variables into account without taking a detour by using a function with string interpolation. Before continuing with the next section, make sure to conduct the exercises. On another note, you haven't used features such as fragments or operation names yet. The application hasn't given any use case for these techniques. But you will use them in another application using Apollo instead of using plain HTTP with axios in the future.
+Finally, the query takes variables into account without detouring into a function with string interpolation. I strongly suggest practicing with the excercises below before continuing to the next section. We've yet to discuss features like fragments or operation names, but we'll soon cover them using Apollo instead of plain HTTP with axios.
 
 ### Exercises:
 
-* explore and add fields to your organization, repository and issues
-  * extend your components to display the additional information
-* read more about {{% a_blank "serving a GraphQL API over HTTP" "http://graphql.org/learn/serving-over-http/" %}}
+* Explore and add fields to your organization, repository and issues
+  * Extend your components to display the additional information
+* Read more about {{% a_blank "serving a GraphQL API over HTTP" "http://graphql.org/learn/serving-over-http/" %}}
 
 {{% chapter_header "React GraphQL Pagination " "react-graphql-pagination" %}}
 
-In the last section, you have already implemented a list field in your GraphQL query. It fitted well into the flow of structuring the query with nested objects and a list along the way with implementing the React components structure where different components were responsible for showing partial results of the query.
+Last section you implemented a list field in your GraphQL query, which fit into the flow of structuring the query with nested objects and a list responsible for showing partial results of the query.
```

Comment:
"... for showing partial results of the query." to "... for showing partial results of the query in React."